disperser/dataapi/metrics_handlers.go

```diff
@@ -142,3 +143,55 @@ func (s *server) getNonSigners(ctx context.Context, intervalSeconds int64) (*[]N
 
 	return &nonSignersObj, nil
 }
+
+func (s *server) getUnsignedBatches(ctx context.Context, intervalSeconds int64) (*UnsignedBatches, error) {
+	nonSigners, err := s.subgraphClient.QueryBatchNonSigningOperatorIdsInInterval(ctx, intervalSeconds)
+	if err != nil {
+		return nil, err
+	}
+
+	var (
+		limit          = 1000
+		skip           = 0
+		totalOfBatches = 0
+	)
+
+loop:
+	for {
+		select {
+		case <-ctx.Done():
+			break loop
+		default:
+			batches, err := s.subgraphClient.QueryBatchesWithLimit(ctx, limit, skip)
```

Comment:
@gpsanant Is this the right way to get total number of batches in the past N seconds?

Reply:
nope, we should see if graphql has a better way to get the count of batches or add a running count to the subgraph

Reply:
Having a running count indexed is certainly nice, but with the current subgraph, is there any better way to query it? If not, we should proceed with this way here. We can revisit later if subgraph has that number indexed.

Reply:
potentially we could look at the registeration/deregistration events here: https://github.com/wmb-software-consulting/eigenda/blob/12b3081fe87edc0d6e8954f63b00db96e2571c20/subgraphs/eigenda-operator-state/schema.graphql#L12 and query batches within, but its pretty complex

Reply:
It seems doable: given N, we want to find total batches that an operator is responsible for
1) Find all register events in past N secs
2) Find all deregister events in past N secs
3) Using the above two, create a list of disjoint time intervals, where the operator is live in the network
4) Query total number batches in each interval, `Ti = Query(Si, Ei)`
5) The sum of all Ti is the answer

In term of implementation:
The step (3) and (5) are easy.
The step (1) and (2) are about querying the two tables, `OperatorDeregistered` and `OperatorRegistered` that already indexed in subgraph. (This is the least unknow part for me, please correct if I'm wrong)
The step (4) is what's already implemented here.

So overall, I think the question is to fetch all register/deregister events for an operator in past N seconds (step 1 and 2).

Reply:
Yep that's straightforward too. It's just complex to put all these together, but if we have bandwidth, let's do it